tests/php/Core/ConvertTest.php

```diff
@@ -224,7 +224,7 @@ public function testArray2JSON()
           'Complicated' => 'Structure'
          )
         );
-        $encoded = Convert::array2json($val);
+        $encoded = json_encode($val);
```

Comment:
These are testing the methods you've replaced, they are deprecated but should stay as they were

Reply:
good catch, sorry